src/integTest/resources/legacy-syntax/transitive-dependencies.gradle

```diff
@@ -9,3 +9,6 @@ repositories {
 dependencies {
   compile 'junit:junit:4.12'
 }
+ossIndexAudit {
+  colorEnabled = false
```

Comment:
Could this param be added to the `README.md` file as documentation?

Reply:
> Could this param be added to the `README.md` file as documentation?

Done. 999e181c